### DISCUSSION TITLE
will v0.4 drop support for serialization

### DISCUSSION THREAD

**Discussion by reececomo** [NateTheGreatt/bitECS#169] [General]
yo! just reading the [docs here](https://github.com/NateTheGreatt/bitECS/tree/rc-0-4-0) --> didn't see any mention of serialization, is that still first-class supported?

**Comment by NateTheGreatt** [comment]
you missed the link :) it's there under the `Documentation` section:
https://github.com/NateTheGreatt/bitECS/blob/rc-0-4-0/README.md#--documentation

direct link:
https://github.com/NateTheGreatt/bitECS/blob/rc-0-4-0/docs/Serialization.md

i also added a networking example:
https://github.com/NateTheGreatt/bitECS/tree/rc-0-4-0/examples/networking

**Comment by reececomo** [comment]
you're right (i swear i ctrl+f'd it, but I just guess I just f'd it 😄)

love it, looks way more flexible than the current one + love the new samples

can one "peek" serialized state without deserializing it directly into an ecs world?

**Reply by NateTheGreatt** [reply]
i didn't add any peeking but it can be added, will add to the todo list :+1:

**Comment by reececomo** [comment]
just realised [this was the docs I read](https://github.com/NateTheGreatt/bitECS/blob/rc-0-4-0/docs/Intro.md), somehow linked to the right doc in this post though 😆
